frameworks/preact/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist"
+  },
+  "include": ["src"],

```

Comment:
FYI the tsconfig.json for React was changed a little for the new `tests` folder: https://github.com/fwouts/previewjs/blob/main/frameworks/react/tsconfig.json

I migrated the React plugin to use [unbuild](https://github.com/unjs/unbuild) to bundle its code, meaning you don't end up with `dist/src/` in the transpiled code. You should be able to use the same config file: https://github.com/fwouts/previewjs/blob/main/frameworks/react/build.config.ts